client/src/wiegand_formats.c

```diff
@@ -704,6 +704,59 @@ static bool Unpack_S12906(wiegand_message_t *packed, wiegand_card_t *card) {
     return true;
 }
 
+
+// if (card->FacilityCode != 217) return false; // Must be 0b11011001 aka 217 for Grinnell?
```

Comment:
There is a few of these comments,   they don't belong in a PR

Reply:
You are right.  I removed the extra comments.
I also uncommented the requirement for the fixed facility code, so it's now required to match that fixed value on both encode and decode.